backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/analysis/model/dsl/ElasticsearchAnalysisComponentDefinitionContext.java

```diff
@@ -7,9 +7,6 @@
 package org.hibernate.search.backend.elasticsearch.analysis.model.dsl;
 
 
```

Comment:
Not relative to the current change. But I think we can take advantage of this PR to fix the too many empty lines between the `package` line and the `interface` line.